### ISSUE TITLE
Menu customization.

### ISSUE THREAD

**Issue by kudilovich** [tgjones/gemini#129]
The out of the box menus are very good, but I would like to remove a couple of features that do not apply in my case. I realize that IMenuBuilder can be implemented and exported, but I have a few questions/requests:
1) When I implement my own MenuBuilder it initializes and finds exports but I can't get anything to display.
2) Because the standard menus are covering a lot of casesI DO need, ideally if there was a method to remove a standard menu item (similar to the way new menu items are added) it would be ideal.
Thanks a lot!

**Comment by tgjones** [comment]
1) What does your custom `MenuBuilder` look like? I just tried this simplest-possible implementation, and it works:

``` csharp
[Export(typeof(IMenuBuilder))]
public class CustomMenuBuilder : IMenuBuilder
{
    public void BuildMenuBar(MenuBarDefinition menuBarDefinition, MenuModel result)
    {
        result.Add(new TextMenuItem(new TextMenuItemDefinition(new MenuItemGroupDefinition(null, 0), 0, "Foo")));
    }
}
```

2) Yes! I definitely want to do this. I'm not sure if it should be a special "exclude" object that you `[Export]`, or a more imperative API that lets you list all the menu items that should be excluded. The latter might be simpler to use, and require less code, but it might also be less flexible. Or perhaps not, I haven't really thought it through yet :)

Pseudo-code for option 1:

``` csharp
[Export]
public static ExcludeMenuItemDefinition ExcludeFileNewMenuItem = new ExcludeMenuItemDefinition(MenuDefinitions.FileNewMenuItem);
```

Pseudo-code for option 2:

``` csharp
var menuCustomiser = IoC.Get<IMenuCustomiser>();
menuCustomiser.ExcludeItems.Add(MenuDefinitions.FileNewMenuItem);
```

Any thoughts on those, or other suggestions?

**Comment by tgjones** [comment]
Fixed via #131.
